.travis.yml

```diff
@@ -1,22 +1,52 @@
 language: cpp
+
+os:
+  - linux
+  - osx
+
+compiler:
+  - gcc
+  - clang
+
+branches:
+  only:
+    - master
+    - travis-osx
+
+before_script:
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew update; fi
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then brew install git qt4; fi
+
 script:
-#- cmake --version
-# Build external tools
-#- pwd
-- mkdir build_external
-- cd build_external
-- cmake ../thirdparty/external
-- make -j4
-- cd ..
-# Build displaz
-#- pwd
-- mkdir build
-- cd build
-- cmake ..
-- make -j4
-- cd ..
+    #- cmake --version
+    # Build external tools
+    #- pwd
+    - mkdir build_external
+    - cd build_external
+    - cmake ../thirdparty/external
+    - make -j4
+    - cd ..
+    # Build displaz
+    #- pwd
+    - mkdir build
+    - cd build
+    - cmake ..
+    - make -j4
+    - cd ..
```

Comment:
Nitpick: if we're going to reindent things, let's make the file self-consistent.  (Ok, I know it wasn't consistent before :) )